examples/platforms/simulation/ble.c

```diff
@@ -26,50 +26,204 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include "platform-simulation.h"
+
+#include <errno.h>
+
+#include <stdio.h>
+#include <stdlib.h>
 #include <openthread/platform/ble.h>
 
+#include "common/code_utils.hpp"
```

Comment:
Do we need this one? Should avoid using it in platform code (this is core internal header).

Reply:
removed